doc/whats_new/v0.23.rst

```diff
@@ -179,6 +179,12 @@ Changelog
   used during `fit`.
   :pr:`16437` by :user:`Jin-Hwan CHO <chofchof>`.
 
+- |Fix| Fixed a bug in :class:`ensemble._BaseStacking` where the
```

Comment:
Instead of pointing to BaseStacking point to StackingClassifier and StackingRegressor since they will be the class used by our users